widget/entry_password_test.go

```diff
@@ -0,0 +1,199 @@
+package widget
```

Comment:
Thanks for thinking about tests - but I think this is just a copy of the original entry_test.go, rather than adding new test checks?

Reply:
Indeed, I've started from a copy of entry_test.go to ensure the password widget behave in the same way except the multiline (covered by `TestPasswordEntry_CursorRow` where is tested that the `Return` key is ignored).
In c4b6b2500a82ad64d3b2c290d961c25b98921c60 a test to cover password obfuscation has been added.

Reply:
I appreciate the idea, but I can't help but feel this is a large duplication for no additional checks.
As PasswordEntry is just an Entry the list of tests is providing no additional coverage.
I think it would be clearer if TestPasswordEntry_CursorRow were named TestPasswordEntry_NewlineIgnored (or similar) so the intent is clear - and keep the new obfuscation test, but the rest are not needed.
Also it should probably be in entry_test.go as this is all testing entry.go.

Sorry to be picky but we need to make sure test are as clear as the main code.

Reply:
No worries, I see your point.
Applied suggestions in 46a26a037f931b94bff8e24cbb4a4474c7d1c132